paimon-core/src/main/java/org/apache/paimon/table/AbstractFileStoreTable.java

```diff
@@ -512,6 +512,11 @@ public void rollbackTo(long snapshotId) {
                 "Rollback snapshot '%s' doesn't exist.",
                 snapshotId);
 
+        // fast return
+        if (new Long(snapshotId).equals(snapshotManager.latestSnapshotId())) {
```

Comment:
I think new Long has a little strange.
Why not:

Long latestSnapshotId = snapshotManager.latestSnapshotId();
if (latestSnapshotId == null) {
    exception or error log
}
if (taggedSnapshot.id() == latestSnapshotId) {
    fast return
}